tailwind.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
-  content: [],
+  content: [
+    "./src/routes/home.jsx",
```

Comment:
Attention, tu peux laisser la ligne telle qu'elle est sur la doc de sorte à ce que ça fonctionne sur tous les fichiers qui sont dans le dossier src/

Reply:
still not ok "./src/**/*.{js,jsx,ts,tsx}",